README.md

```diff
@@ -10,9 +10,7 @@ A bundle to use swarrot inside your Symfony2 application
 
 ## Installation
 
-The recommended way to install this bundle is through
-[Composer](http://getcomposer.org/). Require the `swarrot/swarrot-bundle`
-package into your `composer.json` file:
+The recommended way to install this bundle is through [Composer](http://getcomposer.org/). Require the `swarrot/swarrot-bundle` package into your `composer.json` file:
```

Comment:
What about recommending to use `composer require swarrot/swarrot-bundle` instead of editing the composer.json file?